Logger/LogFactory.cs

```diff
@@ -2,10 +2,32 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
-        {
+        public BaseLogger? CreateLogger()
```

Comment:
```suggestion
        public BaseLogger? CreateLogger(string className)
```
This method should take a parameter of type `string` called `className`.
That parameter should be used when creating your `FileLogger` on line 16-20.